tests/test_views.py

```diff
@@ -23,6 +23,33 @@ def test_study_create(client, user, study_area):
     assert study_area.name in response.content.decode('utf-8')
 
 
+@pytest.mark.django_db
+def test_study_list(client, user):
+    client.force_login(user)
+    url = reverse('studies-list')
+    response = client.get(url)
+
+    assert response.status_code == 200
```

Comment:
Do we have any location or study fixtures that we could require for these `list` tests to make sure that all locations/studies are getting displayed in the list?

Reply:
Yep! I'll make new fixtures for inactive Locations and Studies too.